app/display/model/src/main/java/org/csstudio/display/builder/model/rules/RuleToScript.java

```diff
@@ -370,11 +377,14 @@ else if (pform == PropFormat.FONT)
 
         if (idx > 0)
         {
-            script.append("else:\n");
+            script.append(try_indent).append("else:\n");
             script.append(indent).append(setPropStr).append(formatPropVal(prop, -1, pform)).append(")\n");
         }
         else
-            script.append(setPropStr).append(formatPropVal(prop, -1, pform)).append(")\n");
+            script.append(try_indent).append(setPropStr).append(formatPropVal(prop, -1, pform)).append(")\n");
+
+        script.append("\nexcept PVUtil.PVHasNoValueException:\n");

```

Comment:
Instead of falling back to the original value when the PV has no value, one could argue that it makes sense to fall back to the original value for any error, so just `except:`, no specialization for `PVHasNoValueException`

Reply:
Makes sense. But with a simple `except:` we will lose the stacktrace in the log. Is it possible to catch both Python and Java exceptions **and** print the stacktrace?

Reply:
Similar to issue above: If you consider disconnected as a key state, then just check for that via PV.isDisconnected instead of updating the PVUtil to throw exceptions, adding a `pvInt` just to provoke the exception, and then having to catch that exception.

Reply:
If it is okay to fall back to the original value in case of any exception that is fine by me; I'll revert PVUtil.java
I will also print the stacktrace of the exception we caught.